### DISCUSSION TITLE
Any Ideas Or Suggestions?

### DISCUSSION THREAD

**Discussion by FadTheChad** [FadTheChad/FadBot#6] [Ideas]
# Hey There
Basically so uhhh, i made this discussion channel thing so that i could hear from you guys on how bad/shit the bot is and how i could make it less bad/shit.

Basically just submit any of your suggestions/ideas here.

## Other Ways Of Reaching Me
I might not able to see this discussion channel that much so you can also spam-ping me on discord aswell :trollface:

**Comment by pranshu05** [comment]
no

**Reply by FadTheChad** [reply]
Good suggestion will hang it on the board

**Comment by pranshu05** [comment]
I think adding automod features will be amazing

**Reply by FadTheChad** [reply]
Yep blacklist system coming soon :)
